File: OCH_Tao/ABC/B/221.py
S = list(input())
T = input()
if "".join(S)==T:
  print("Yes")
else:
  for i in range(len(S)-1):
    S[i],S[i+1]=S[i+1],S[i]
    if "".join(S)==T:
      print("Yes")
      break
    else:
      S[i],S[i+1]=S[i+1],S[i]
  else:
    print("No")

File: OCH_Tao/ABC/B/221.js
const [S, T] = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
if (S === T) {
  console.log("Yes");
} else {
  let X = [...S];
  for (let i = 0; i < S.length - 1; i++) {
    [X[i], X[i + 1]] = [X[i + 1], X[i]];
    if (X.join("") === T) {
      console.log("Yes");
      process.exit();
    } else {
      [X[i], X[i + 1]] = [X[i + 1], X[i]];
    }
  }
  console.log("No");
}